tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.15-alpine AS builder
+
+ENV GO111MODULE=on \
```

Comment:
If we agree to go 1.16 we don't need to specify this as it's on by default.
https://blog.golang.org/go116-module-changes